streaming/base/storage/upload.py

```diff
@@ -222,23 +229,29 @@ def upload_file(self, filename: str):
         Args:
             filename (str): File to upload.
         """
-        local_filename = os.path.join(self.local, filename)
-        remote_filename = os.path.join(self.remote, filename)  # pyright: ignore
-        obj = urllib.parse.urlparse(remote_filename)
-        logger.debug(f'Uploading to {remote_filename}')
-        file_size = os.stat(local_filename).st_size
-        with tqdm.tqdm(total=file_size,
-                       unit='B',
-                       unit_scale=True,
-                       desc=f'Uploading to {remote_filename}',
-                       disable=(not self.progress_bar)) as pbar:
-            self.s3.upload_file(
-                local_filename,
-                obj.netloc,
-                obj.path.lstrip('/'),
-                Callback=lambda bytes_transferred: pbar.update(bytes_transferred),
-            )
-        self.clear_local(local=local_filename)
+
+        @retry(num_attempts=self.retry)
+        def _upload_file():
+            print('number of retries')
```

Comment:
did you mean to print the number of retries here? otherwise may want to delete the line.

Reply:
Removed since I accidentally pushed it since I added for testing.